moped-editor/src/views/projects/projectView/ProjectWorkActivity/ProjectWorkActivityTable.js

```diff
@@ -0,0 +1,208 @@
+import { useMemo, useState, useCallback } from "react";
+import { useParams } from "react-router-dom";
+import { useQuery, useMutation } from "@apollo/client";
+import Box from "@mui/material/Box";
+import CircularProgress from "@mui/material/CircularProgress";
+import { DataGrid } from "@mui/x-data-grid";
+import EditOutlinedIcon from "@mui/icons-material/EditOutlined";
+import DeleteOutlineIcon from "@mui/icons-material/DeleteOutline";
+import IconButton from "@mui/material/IconButton";
+import Link from "@mui/material/Link";
+import OpenInNew from "@mui/icons-material/OpenInNew";
+import ApolloErrorHandler from "src/components/ApolloErrorHandler";
+import WorkActivityToolbar from "./ProjectWorkActivityToolbar";
+import ProjectWorkActivitiesDialog from "./ProjectWorkActivityDialog";
+import { getUserFullName } from "src/utils/userNames";
+import {
+  WORK_ACTIVITY_QUERY,
+  DELETE_WORK_ACTIVITY,
+} from "../../../../queries/funding";
+import { currencyFormatter } from "src/utils/numberFormatters";
+
+/** Hook that provides memoized column settings */
+const useColumns = ({ deleteInProgress, onDeleteActivity, setEditActivity }) =>
+  useMemo(() => {
+    return [
+      {
+        headerName: "",
+        field: "_edit",
```

Comment:
I was messing around with trying to hide columns and remembering how datagrid works, and I think it would be good to add 
`        hideable:false,
        filterable:false,
        sortable: false,` to this _edit field

Reply:
done—thank you!